src/controller/java/BUILD.gn

```diff
@@ -75,10 +81,7 @@ android_library("java") {
 
   deps = [ "${chip_root}/third_party/java_deps:annotation" ]
 
-  data_deps = [
-    ":jni",
-    "${chip_root}/build/chip/java:shared_cpplib",
```

Comment:
Why removed java:shared_cpplib?

Reply:
if you see line 139, you will see it appear there, here it is duplicate...